ext/socket/raddrinfo.c

```diff
@@ -498,9 +524,6 @@ rb_getaddrinfo(const char *hostp, const char *portp, const struct addrinfo *hint
         errno = err;
         return EAI_SYSTEM;
     }
-    pthread_detach(th);
-
-    rb_thread_call_without_gvl2(wait_getaddrinfo, arg, cancel_getaddrinfo, arg);
```

Comment:
Was this deleted on purpose? it seems important to me! (it's the bit that actually waits on the condvar for the lookup to be done, right?)

Reply:
Nah, just a bad rebase I think, let me check.

Reply:
Ok, fixed it. It should have been replaced by `fork_safe_async_getaddrinfo(arg);`